submissions/sprint_b/test_helpers.py

```diff
@@ -0,0 +1,103 @@
+import unittest
```

Comment:
MIssing module, class, and method docstrings describing thier respective purpose.